kaizen/evaluator.py

```
@@ -2,117 +2,270 @@
-    def __init__(self, provider: str, logger: Optional[TestLogger] = None):
+    def __init__(self, provider: str = "google", logger: Optional[TestLogger] = None):
         self.provider = provider
         self.config = Config()
-        self.api_key = self.config.get_api_key(provider)
+        # Try both 'google' and 'gemini' as provider names for the API key
+        self.api_key = self.config.get_api_key(provider) or self.config.get_api_key("google")
         if not self.api_key:
-            raise ValueError(f"API key not found for provider: {provider}. Please add it to your .env file.")
+            raise ValueError(f"API key not found for provider: {provider}. Please add it to your .env file as GOOGLE_API_KEY.")
         
         self.logger = logger or TestLogger()
```

Pylint (3.3.7)

[error] No value for argument 'test_name' in constructor call

(E1120)